app/views/draws/intent_report.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= provide(:title, "Intent report - " + @draw.name) %>
```

Comment:
I think we're better off with `"#{@draw.name} - Intent report"` and similar for all the other sub-views.